index.html

```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <h1 class="titulo-libro"><strong>Título del libro</strong> <p class= "titulo-genero"> (género)</p></h1> <br>
```

Comment:
Try to write the classes (and every text that it's not directed towards the user) in English